src/Components/Filters.js

```diff
@@ -60,6 +97,16 @@ const styles = StyleSheet.create({
     backgroundColor: '#eee',
     borderColor: '#d5d5d5',
   },
+  input: {
+    marginBottom: 30,
+    padding: 10,
+    paddingHorizontal: 20,
+    fontSize: 16,
+    color: '#444',
+    borderBottomWidth: 1,
+    borderColor: '#ddd',
+    backgroundColor: '#F5F5F5',
```

Comment:
We should probably start using a central file to manage colors, otherwise this is going to be one hell of a rainbow :)

Reply:
I did this in another pull request: https://github.com/UltimateApp/UltimateApp/pull/49